src/postgres/tree/PostgresSchemaTreeItem.ts

```diff
@@ -3,22 +3,23 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { Schema, Table } from 'pg-structure';
-import * as _ from 'underscore';
+import { Schema } from 'pg-structure';
 import * as vscode from 'vscode';
 import { AzureParentTreeItem, ISubscriptionContext } from 'vscode-azureextensionui';
 import { getThemeAgnosticIconPath } from '../../constants';
-import { PostgresTableTreeItem } from './PostgresTableTreeItem';
+import { PostgresTablesTreeItem } from './PostgresTablesTreeItem';
 
 export class PostgresSchemaTreeItem extends AzureParentTreeItem<ISubscriptionContext> {
     public static contextValue: string = "postgresSchema";
     public readonly contextValue: string = PostgresSchemaTreeItem.contextValue;
-    public readonly childTypeLabel: string = "Table";
     public readonly schema: Schema;
```

Comment:
Probably want to set `autoSelectInTreeItemPicker` just like [here](https://github.com/microsoft/vscode-azurestorage/blob/3ea68d7f2f868555c48b04d0b5395544ed4b1668/src/tree/StorageAccountTreeItem.ts#L49)

We don't want the "Tables" tree item to show up in the command palette